Packs/CloudIncidentResponse/Scripts/ExtractIndicatorsCloudLogging/ExtractIndicatorsCloudLogging.py

```diff
@@ -0,0 +1,65 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+import json
+
+
+def extract_aws_info(event):
+    arn = event['userIdentity']['arn']
```

Comment:
Please change all usage of ```[]``` to using ```get``` method.

Reply:
Done